projects/setuptools.cmake

```diff
@@ -0,0 +1,15 @@
+#add_custom_target(setuptools)
+#
+#add_custom_command(COMMAND ${PYTHON_EXECUTABLE_PREFIXED} -m pip install -U setuptools
+#                   TARGET setuptools)
+
+ExternalProject_Add(setuptools
```

Comment:
Thanks, I was looking for this for a while, but ended up with adding a recipe.
I remembered you already mentioned this in another PR months before, but also couldn't this post again.